docs/en/fluent.md

```diff
@@ -0,0 +1,64 @@
+# Fluent support
+
+* [Localising your Partial Cache](#localising-your-partial-cache)
+    * [Use configuration (ideal)](#use-configuration-ideal)
+    * [Use extension (less ideal, but still fine)](#use-extension-less-ideal-but-still-fine)
+    * [Add it to your Partial Cache manually](#add-it-to-your-partial-cache-manually)
+
+We have considered Fluent for this module, however, the level of configuration available in Fluent makes it difficult
+for us to prescribe a solution at this time.
+
+Fluent itself provides us with the mechanism we require to have a separate cache for each frontend Locale that our users
+browse, however, it is important to note that with our "out of the box" implementation, when any Localisation for
+a `DataObject` is updated, it will invalidate the cache for **all** Localisations of that particular `DataObject`.
+
+This does mean that you will be invalidating the cache for Locales that might not have been updated, but we feel that
+this is acceptable in these early stages of this module's life. The important thing is that the caches **will be
+unique** for each Locale that your users browse, and you **will not** have data "leak" from one Locale to another.
+
+## Localising your Partial Cache
+
+This is not specific to Keys for Cache, and is generally recommended for any project using Fluent.
```

Comment:
[citation needed] 😂

Reply:
I may have taken some liberties with this statement....

They **do** recommend you add the CurrentLocale, but I don't think we were aware of this config mechanism when we wrote that haha.